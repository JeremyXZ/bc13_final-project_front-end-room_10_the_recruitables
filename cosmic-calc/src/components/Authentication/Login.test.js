import React from "react";
import "@testing-library/jest-dom";
import { screen, render } from "@testing-library/react";
import userEvent from "@testing-library/user-event";

import LoginButton from "./Login";

describe("<LoginButton/>", () => {
  
  it("successfully renders the component, matches snapshot", () => {
    const { asFragment } = render(<LoginButton />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("button text contains 'log in", () => {
    render(<LoginButton />);
    const button = screen.getByRole("button");   
    // expect(button).toHaveTextContent(mockedProps.text);
    expect(button).toHaveTextContent(/log in/i);
  });

 
});