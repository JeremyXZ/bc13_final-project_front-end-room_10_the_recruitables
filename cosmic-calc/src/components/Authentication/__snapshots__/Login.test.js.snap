// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginButton/> successfully renders the component, matches snapshot 1`] = `
<DocumentFragment>
  <button>
    Log In
  </button>
</DocumentFragment>
`;
